torch/csrc/autograd/python_fft_functions.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <Python.h>
+#include <pybind11/pybind11.h>
```

Comment:
we do this for nn?  We should limit our header includes to what's needed by the header, but okay.

Reply:
We do: 

https://github.com/pytorch/pytorch/blob/3c6fae6567230da0e08788b968cfb466ec629d65/torch/csrc/autograd/python_nn_functions.h#L4

I agree with you but it seems like a separate issue.

Reply:
Note from review: trying building without these headers here.

Reply:
Try forward declaring the PyObject instead of including Python.h.